coding_style/README.adoc

```diff
@@ -161,6 +165,9 @@ Rationale:: https://yaml.org/type/bool.html[YAML 1.1] allows all variants wherea
   your task will likely need options such as `changed_when:` and maybe `check_mode:`).
 * Anytime `command` or `shell` modules are used, add a comment in the code with justification to help with future maintenance.
 * Use the `| bool` filter when using bare variables (expressions consisting of just one variable reference without any operator) in `when`.
+* No single task file should contain more than 30 tasks.  Complex task files should be broken down into discrete parts.
```

Comment:
Why 30? That seems arbitrary without an explanation to why specifically 30. What is meant by "discrete parts"? I have sophisticated automation play with 50 tasks. Is the 30 task an include to another task file that contains the other 20?

Reply:
Because it must be a number. Why black has a line limit of 88. They had to make a decision.

TBH, I do think that 30 is reasonable and we will find-out how big is the impact while we have a PR to implement it in linter, so we could tune it (before merging). 20 seems dangerously low for me, while 50 likely too big. Maybe setting it to "42", to play safe? ;)

Keep in mind that users might be able to disable the rule per file. Still, I would be (softly) against adding a configurable option.